scripts/connection.php

```diff
@@ -0,0 +1,11 @@
+<?php
+$config = require ($_SERVER['DOCUMENT_ROOT'] . "../config.php");
```

Comment:
```suggestion
$config = require (__DIR__ . "/../config.php");
```

I don't know whether `$_SERVER['DOCUMENT_ROOT']` is allowed to end in a slash or not, so this might result in a double slash… The `..` is wrong regardless.

Reply:
Just thought about it, `$_SERVER['DOCUMENT_ROOT']` doesn't work if the files are not in the root directory.